docs/guides/src/main/operator/basic-deployment.adoc

```diff
@@ -0,0 +1,174 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+<#import "/templates/options.adoc" as opts>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Basic Keycloak Deployment"
+summary="How to install Keycloak using the Operator on Kubernetes or OpenShift">
+
+== Basic Keycloak Deployment
+In this guide we will show how to have a basic Keycloak Deployment on Kubernetes or OpenShift using the Operator.
+We assume that the Operator is correctly installed and running in the cluster namespace.
+
+=== Pre-requisites
+
+* Database
+* Hostname
+* TLS Certificate and associated keys
+
+==== Database
+
+A database should be available and accessible from the cluster namespace where you want to install Keycloak.
+Please refer to <@links.server id="db"/> for the list of supported databases.
+The Keycloak Operator does not manage the database and you need to provision it yourself, we suggest to verify your cloud provider offering or use a database Operator such as Crunchy[https://access.crunchydata.com/documentation/postgres-operator/latest/].
```

Comment:
I noticed the link doesn't render correctly:
![Screenshot 2022-03-31 at 17 01 19](https://user-images.githubusercontent.com/13906855/161086817-a6129f5c-88f8-4c6a-befc-6d371ab9fae7.png)

We probably need something like:
```suggestion
The Keycloak Operator does not manage the database and you need to provision it yourself, we suggest to verify your cloud provider offering or use a database Operator such as https://access.crunchydata.com/documentation/postgres-operator/latest/[Crunchy].
```
Ref.: https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/